File: my_solution.js
class ListNode {
    constructor(value = 0, next = null) {
        this.value = value
        this.next = next
    }
}

/* 

hold a sum variable
hold a current variable
while current variable isn't null, then add the value to the sum
return sum

1 > 4 > 5
        c
*/

function sumElements(head) {
   if(!head) return 0
    
    let sum = 0
    let curr = head
    while(curr){
        sum += curr.value
        curr = curr.next
    }
    return sum
}

// Test Cases
var LL1 = new ListNode(1, new ListNode(4, new ListNode(5)))
console.log(sumElements(null)) // 0
console.log(sumElements(LL1)) // 10
console.log(sumElements(new ListNode(1))) // 1

File: my_solution.py
class ListNode:
    def __init__(self, value = 0, next = None): 
        self.value = value
        self.next = next
        
def sumElements(head: ListNode) -> int:
    if not head:
        return 0
    
    sum = 0
    curr = head
    while curr:
        sum += curr.value
        curr = curr.next
        
    return sum
    

# Test Cases
LL1 = ListNode(1, ListNode(4, ListNode(5)))
print(sumElements(None)) # 0
print(sumElements(LL1)) # 10
print(sumElements(ListNode(1))) # 1
